src/open_samus_returns_rando/files/schema.json

```diff
@@ -114,21 +114,31 @@
             }
         },
         "elevators": {
-            "type": "array",
-            "items": {
-                "type": "object",
-                "properties": {
-                    "teleporter": {
-                        "$ref": "#/$defs/actor_reference_with_layer"
-                    },
-                    "destination": {
-                        "$ref": "#/$defs/actor_reference"
+            "type": "object",
+            "description": "A dictionary of dictionaries mapping scenario and elevator actor to a connecting elevator",
```

Comment:
Please add a `"default": {},` to it. 
Otherwise all patcher.json files break which is unecessary. (And it would also require to add an empty field from randovania etc....and to be totally nitpicky, it would technically be also version 2.0.0 then 😄)

Reply:
Will do